packages/backend/localEntryNodePeerId.json

```diff
@@ -1 +1,5 @@
\ No newline at end of file
+{
+  "id": "12D3KooWHgLdRMqkepNiYnrur21cyASUNk1f9NZ5tuGa9He8QXNa",
+  "privKey": "CAASpgkwggSiAgEAAoIBAQCk3FjDvLsRyvIDrDTaJ8AG1ygyBS27tXizjDRAsgPmGLIGin7Ys/EjDe8G8CU65Efngc0I9nKPfOF/ZgvhGoMKgiVe5jF3feA8pV3WCu9ZVYTCWcT1Yzg6KuQnqFzEhHg5eZBTImak4syoSGlnHICl6ltsLqzndBIK3GqCQo4GN31YNGa1kzLmX54DXfV56iXRNGMFRrr62ovVjdmKgXJhZ5hU9IHxA3IPzH9oajS6fd5U/NDu1CGQhznIbUR15CLN1NFW3uyK+ByqjarEKTxs7RDNuqR2VOoIe/ovP/TCO6QmZIff6+EOrpOHg0vvQ1RCOKoJxZrpXnyOrprGTVATAgMBAAECggEAbfyfr77bI0y9V5vtCtpYL3gd1flEgKXaaBcSyVK6bSlupBSD+fla2CM0o/FVNUwVeKtNajxZQ8dx/ffL1xtD75/SZNS4jdrJ3kwJJYLYGkaGss8HKxtN6OiynI6jjWw/dUDX+87Iohq5pU/iGiuW3JpU8gDV0/jJDLvuxwa2EAikfTIzBOs4GnZ28AMvmAt5aLevL6UpaG3r4cx+7zLEr8VHu0jEXY5c/aO0dA+T31jTKFBjuk9rShGeJ39VCml8lgIXsqTDeqxiTgFLVm3GFQkE1bFN9nsUsJA4G/bhIlaFBut12ENpa/CC758zAYH7UDQiw4hafyJzJv6utHRNgQKBgQDbckfJGcMJbJ3YlEbaChlnMYff6LAAj/FAxEioRJx6GwgMm8RBdzVme1UOkefe7zX+nKulsll2PHcc97H2VXMGavHxB/tH4rOVwY0M28Dj3LkjPN6pV3ouzozHW2uuiJx0gXvxkks37CXt6yu0IIGEYt2CnhxV7Pz5mOh+bOJiQQKBgQDAUmcIF/ZIuFCRR8FfxQX/2oc40itOMIdcJ3PsUeqjX6ZOkG252tMSTrAd9g/Pl1GsG2MUR8dO6NcQydZhsJ6BSgcRZHxcCl6lfp/fOPLriyS8I6f1/iXcvJ6Tl7itziwP6EMYgl/IZhVzHzUY5r1Jf4kiQiL14yJf/btZ1pA1UwKBgBpkcjHDgWt+vrUvkB3lHQ0GLftF9vT6Wt+ZO8vRBKjC24L4sO/WsHt01dP8xFBRoMLQgOxauNu5hfHb8HFZ8Y2SCysb9KK+py6ZqqcjPt2ozDNqvLTNKoq6mL/ZROgm0fpJIGj7WUSOiDlHEWiGiE0tvspYzbuf1wUF2OBrpAaBAoGAbhn+sCx/q4wRW8q3kutJjvr9NPNlJWM06vYPMHiJGrS0kf+vQZpcAfwYaV6oGJEpiZ04MLETEkLMdOkIGW8ynxPRTgy4AT1Sh+4g+nsw3tYO1sKW8oZoCKHXKunOKXqB0S2H0+03hpOirgBstm/KZ9xilsFEIH7C/C9XZYwnjM8CgYBq6sBzj4PfDAGqgNJbWyY6Ep1at4luOl7yhY3gX8UK2p3Ih/dfl5d1FdIEtEwz7RziVKm0JazHHWmXNPX/aZHUOKfyNCt/s18Lt/7LiZsdvN6W7BpL3fyhpEUV+9XRorI49eVo3NNiNbDVxCKmZzYgWqHgd9PYwaHpZ4M0YJF4eg==",
```

Comment:
Are these used for e2e tests? Is it a security issue that these are in the repo in the first place?

Reply:
Any static references to keys or peer IDs are purely test related.